src/Modules/OrchardCore.Commerce/Startup.cs

```diff
@@ -256,18 +257,15 @@ public class DeploymentStartup : StartupBase
 {
     public override void ConfigureServices(IServiceCollection services)
     {
-        services.AddTransient<IDeploymentSource, SiteSettingsPropertyDeploymentSource<RegionSettings>>();
-        services.AddScoped<IDisplayDriver<DeploymentStep>>(serviceProvider =>
-            {
-                // It's the IStringLocalizer.
-#pragma warning disable SA1312 // Variable names should begin with lower-case letter
-                var T = serviceProvider.GetService<IStringLocalizer<DeploymentStartup>>();
-#pragma warning restore SA1312 // Variable names should begin with lower-case letter
-                return new SiteSettingsPropertyDeploymentStepDriver<RegionSettings>(
-                    T["Region settings"],
-                    T["Exports the region settings."]);
-            });
-        services.AddSingleton<IDeploymentStepFactory>(new SiteSettingsPropertyDeploymentStepFactory<RegionSettings>());
+        services.AddDeployment<SiteSettingsPropertyDeploymentSource<RegionSettings>, SiteSettingsPropertyDeploymentStep<RegionSettings>>();
+        services.AddScoped(ImplementationFactory);
+    }
+
+    [SuppressMessage("Naming Rules", "SA1312:Variable names should begin with lower-case letter", Justification = "It's the IStringLocalizer.")]
+    private IDisplayDriver<DeploymentStep> ImplementationFactory(IServiceProvider serviceProvider)
+    {
+        var T = serviceProvider.GetService<IStringLocalizer<DeploymentStartup>>();
```

Comment:
Why not just name it `localizer`?

Reply:
It was already like that above, I just moved the code around. Also I think `localizer` is not picked up by the PO Extractor. (although it really should, let me check if maybe there is already an issue for that).